code/modules/dwarven/dwarven_items.dm

```diff
@@ -0,0 +1,167 @@
+/obj/item/dwarven_guide
+	name = "Brokering 101: Dwarven guide"
+	desc = "An ancient tome written dwarven"
+	icon = 'icons/obj/library.dmi'
+	icon_state = "book1"
+	w_class = 2
+
+/obj/item/dwarven_guide/attack_self(mob/living/carbon/human/user)
+	if(!user.can_read(src) || !user.dna.check_mutation(DWARVEN))
+		return FALSE
+	to_chat(user, "<span class='notice'>You flip through the pages of the book, quickly and conveniently learning every language in existence. Somewhat less conveniently, the aging book crumbles to dust in the process. Whoops.</span>")
+	user.grant_all_languages()
+	new /obj/effect/decal/cleanable/ash(get_turf(user))
+	qdel(src)
+
+/obj/item/twohanded/war_hammer
+	name = "dwarven warhammer"
+	desc = "A very heavy warhammer, used to dent skulls unless they are already dented."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "greyscale_dwarven_warhammer0"
+	//item_state = "greyscale_dwarven_warhammer0"
+	lefthand_file = 'icons/mob/inhands/weapons/hammers_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/hammers_righthand.dmi'
+	flags_1 = CONDUCT_1
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
+	force = 14
+	force_unwielded = 14
+	force_wielded = 20
+	armour_penetration = 20
+	w_class = WEIGHT_CLASS_BULKY
+	custom_materials = list(/datum/material/iron = 20000)
+	attack_verb = list("smashed", "dented", "bludeoned")
+	hitsound = 'sound/weapons/slam.ogg'
+	sharpness = IS_BLUNT
+
+/obj/item/twohanded/war_hammer/update_icon_state()
+	icon_state = "greyscale_dwarven_warhammer[wielded]"
+
+
+/obj/item/hatchet/dwarven/axe
+	name = "dwarven hand axe"
+	desc = "A very sharp axe blade made of greatest dwarven metal."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "greyscale_dwarven_axe"
+	item_state = "greyscale_dwarven_axe"
+	lefthand_file = 'icons/mob/inhands/weapons/axes_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/axes_righthand.dmi'
+	flags_1 = CONDUCT_1
+	material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
```

Comment:
this could be put in /obj/item/hatchet/dwarven since it doesnt exist right now and its shared between the two things